File: Miscellaneous/Golang/171. Excel Sheet Column Number.go
package main

import "fmt"

func main() {

	columnTitle := "AA"
	result := titleToNumber(columnTitle)
	fmt.Println(result)
}

func titleToNumber(columnTitle string) int {
	ans := 0

	for i := 0; i < len(columnTitle); i++ {
		ans = ans*26 + int(columnTitle[i]) - 64
	}

	return ans
}

File: Miscellaneous/Python/171. Excel Sheet Column Number.py
class Solution:
    def titleToNumber(self, columnTitle: str) -> int:
        ans = 0
        for char in columnTitle:
            ans = ans*26 + ord(char) - 64
        
        return ans

obj = Solution()
columnTitle = "AZ"
ans = obj.titleToNumber(columnTitle)
print(ans)